onnxruntime/core/optimizer/graph_transformer_utils.cc

```diff
@@ -151,6 +153,8 @@ std::vector<std::unique_ptr<GraphTransformer>> GenerateTransformers(TransformerL
       transformers.emplace_back(onnxruntime::make_unique<AttentionFusion>(cpu_cuda_execution_providers));
       transformers.emplace_back(onnxruntime::make_unique<EmbedLayerNormFusion>(cpu_cuda_execution_providers));
 
+      transformers.emplace_back(onnxruntime::make_unique<BiasDropoutFusion>(cpu_cuda_execution_providers));
+      transformers.emplace_back(onnxruntime::make_unique<MatmulTransposeFusion>(cpu_cuda_execution_providers));
```

Comment:
MatmulTransposeFusion is a duplication of MatMulScaleFusion at line 160. 
MatmulTransposeFusion should be deprecated, and we should fix MatMulScaleFusion and use it.

Reply:
https://msdata.visualstudio.com/Vienna/_sprints/taskboard/ONNX%20Training/Vienna/Cobalt?workitem=1028960

Reply:
adding a TODO for now to fix and combine the transforms